index.bs

```diff
@@ -7287,7 +7287,7 @@ partial dictionary MLOpSupportLimits {
   [  12  12.25  12.75  13.25  13.75  14.25  14.75  15  ]
   ```
 
-  This has the convenient properties that the sampling is evenly distributed and the corner values are aligned when up-sampling.
+  This has the convenient properties that the sampling is evenly distributed, symmetric, robust to image mirroring, and the corner values are aligned (when up-sampling).
```

Comment:
> (when up-sampling)

When downsampling too, right? Can we just remove the parenthesized bit?

Reply:
Done in d26a151